extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -517,7 +662,7 @@ export class AmpStoryQuiz extends AMP.BaseElement {
    */
   updateQuizOnDataRetrieval_() {
     let selectedOptionKey;
-    this.responseData_.responses.forEach(response => {
+    this.responseData_['responses'].forEach(response => {
```

Comment:
Sorry I missed this in the previous review: in the line below `if (selectedOptionKey === undefined) {` can you add a `return` statement as well? The `dev().error()` will get stripped from the production code, we need a way to stop the execution.